src/DotLiquid.Tests/Embedded/golden_rules.json

```diff
@@ -0,0 +1,227 @@
+{
+    "skipped_groups": [
+        "liquid.golden.echo_tag", // not yet implemented
+        "liquid.golden.liquid_tag", // not yet implemented
+        "liquid.golden.render_tag", // not yet implemented
+        "liquid.golden.inline_comment_tag", // not yet implemented
+        "liquid.golden.replace_last_filter", // not yet implemented
+        "liquid.golden.sum_filter", // not yet implemented
+        "liquid.golden.truncatewords_filter" // implemented as truncate_words so these will fail for now
+    ],
+    "failing_tests": [
+        // Parser Errors
+        "liquid.golden.assign_tag - assign with quoted notation and extra whitespace",
+        "liquid.golden.identifiers - trailing question mark in for loop target",
+        "liquid.golden.identifiers - trailing question mark output",
+        "liquid.golden.lstrip_filter - left and right padded",
+        "liquid.golden.lstrip_filter - left padded",
+        "liquid.golden.lstrip_filter - right padded",
+        "liquid.golden.newline_to_br_filter - reference implementation test 1",
+        "liquid.golden.newline_to_br_filter - reference implementation test 2",
+        "liquid.golden.newline_to_br_filter - string with newlines",
+        "liquid.golden.rstrip_filter - left and right padded",
+        "liquid.golden.rstrip_filter - left padded",
+        "liquid.golden.rstrip_filter - right padded",
+        "liquid.golden.strip_filter - left and right padded",
+        "liquid.golden.strip_filter - left padded",
+        "liquid.golden.strip_filter - right padded",
+        "liquid.golden.strip_newlines_filter - reference implementation test 1",
+        "liquid.golden.strip_newlines_filter - reference implementation test 2",
+        // Default filter allow_false attribute is not yet implemented
+        "liquid.golden.default_filter - allow false",
+        "liquid.golden.default_filter - allow false from context",
+        "liquid.golden.default_filter - false keyword argument before positional",
+        "liquid.golden.default_filter - true keyword argument before positional",
+        "liquid.golden.default_filter - zero is not falsy with allow_false",
+        "liquid.golden.default_filter - zero is not falsy with allow_false",
+        "liquid.golden.default_filter - render a default given a literal false with 'allow false' equal to false",
+        "liquid.golden.default_filter - render a default given a literal false with 'allow false' equal to true",
+        // DotLiquid #457
+        "liquid.golden.date_filter - seconds since epoch format directive",
```

Comment:
This is failing due to timezone. As such CI doesn't pick up on this error since it operates in UTC. New group of flaky tests?